### PR TITLE
Consent status custom view (EXPOSUREAPP-3737)

### PR DESCRIPTION
**Pull request by AlexanderAlferov**
Consent status custom view

Screenshot 2020-11-24 at 17 00 02

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/view_consent_status.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/layout/view_consent_status.xml:1]: Can you use a ConstraintLayout to flatten the view hierarchy?

Comment (AlexanderAlferov) [reply Corona-Warn-App/src/main/res/layout/view_consent_status.xml:1]: Done ✅

**Review by chiljamgossow — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/layout/view_consent_status.xml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/layout/view_consent_status.xml:2]: This currently uses 3-nested ConstraintLayouts, please make it use only one.

File: Corona-Warn-App/src/main/res/layout/view_consent_status.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_mega_tiny"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.0">
+
+            <ImageView
+                style="@style/icon"
+                android:layout_width="@dimen/icon_size_risk_details_behavior"
```
Comment (d4rken) [inline Corona-Warn-App/src/main/res/layout/view_consent_status.xml:40]: This custom view has no relationship with risk details behavior. I would prefer if we either add a new dimension (if there are other similar views with such icons as this one, or just hardcode it here as it a custom view.

**Review by d4rken — CHANGES_REQUESTED** [review]

**Review by d4rken — DISMISSED** [review]

**Review by chiljamgossow — DISMISSED** [review]

**Review by BMItr — DISMISSED** [review]
nice custom view :ok_hand:
just resolve last conflict and then give it a go.

**Review by BMItr — APPROVED** [review]

**Review by chiljamgossow — APPROVED** [review]

**Pull request merged**